app/views/clients/messages.ejs

```diff
@@ -46,7 +46,9 @@
         <div class="message <% if (showWarning) { %>warning<% } %> <%= message.inbound ? 'inbound' : 'outbound' %>"
               <% if (messageIndex == messages.length - 1) { %>id="lastMessage"<% } %>>
           <div class="content">
-            <%= message.content %>
+            <% message.content.split('\n').forEach((line) => { %>
+              <%= line %><br/>
+            <% }) %>
```

Comment:
This kind of thing is begging to be made into a helper method. [This is how rails does it.](http://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-simple_format) This could be done subsequently though, no need to do it now.